tests/system/all_sets.cfg

```diff
@@ -168,11 +168,30 @@ print_statements  = True
 
 [#]
 sets = ["enso_diags"]
-plot_type = "scatter"
-case_id = "TREFHT"
+case_id = 'TREFHT-response-map-start-yrs'
 debug = True
+diff_colormap = "BrBG"
+output_format_subplot = ["png"]
+regions = ["20S20N"]
 results_dir = "all_sets_results_test"
+seasons = ["ANN"]
+test_start_yr='2012'
```

Comment:
@chengzhuzhang Here we test using the following parameter combination: `test_start_yr`, `ref_start_yr`, and `end_yr`. Should we test others or do you think this is sufficient?

Reply:
I think most use will be having test_start_yr test_end_yr ref_start_yr ref_end_yr. Maybe we should have this set test out instead?